src/redux/books/books.js

```diff
@@ -0,0 +1,35 @@
+const initialState = [];
+
+const ADDBOOK = 'ADDBOOK';
+const REMOVEBOOK = 'REMOVEBOOK';
```

Comment:
- [ ] nice work. please ensure that the value of each action type follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so for add-book, it will be something like:       const ADD_BOOK = bookstore/books/ADD_BOOK